jenkins/ci.opensuse.org/openstack-trackupstream.yaml

```diff
@@ -123,6 +124,9 @@
             OBS) OSCAPI="https://api.opensuse.org"
                 OSC_BUILD_ARCH=x86_64
                 case $OBS_PROJECT in
+                    Cloud:OpenStack:Pike)
```

Comment:
Should this be Pike* to include the staging project, the way the other cases are on line 130:

Reply:
Done